Allow the pip package provider to work for the default  
python3 version

Index: lib/puppet/provider/package/pip3.rb
--- lib/puppet/provider/package/pip3.rb.orig
+++ lib/puppet/provider/package/pip3.rb
@@ -15,6 +15,6 @@ Puppet::Type.type(:package).provide :pip3,
   has_feature :installable, :uninstallable, :upgradeable, :versionable, :install_options, :targetable
 
   def self.cmd
-    ["pip3"]
+    ["pip${MODPY_DEFAULT_VERSION_3}"]
   end
 end
